src/directives/apMesaRows.js

```diff
@@ -129,6 +129,7 @@ angular.module('apMesa.directives.apMesaRows',[
     }, function(e) {
       scope.transientState.getDataPromise = null;
       scope.transientState.loadingError = true;
+      scope.errorData = e;
```

Comment:
Also, rename this to `getDataError`, that way other errors can be similarly named.